cli/mfg.c

```diff
@@ -404,12 +404,56 @@ static int image_select(int argc, char **argv)
 	return ret;
 }
 
+static int boot_resume(int argc, char **argv)
+{
+	const char *desc = "Resume device boot process (BL1 and BL2 only)\n\n"
+			   "A normal device boot process includes BL1, "
+			   "BL2 and Main Firmware boot phases. In the case "
+			   "when boot process is paused at BL1 or BL2 phase "
+			   "(due to boot failure or BOOT_RECOVERY PIN[0:1] "
+			   "being set to LOW), sending this command requests "
+			   "device to try resuming normal boot process.";
```

Comment:
So after this command the device will be back into normal firmware? That's probably the important thing to mention.

How does this interact with the OS? Does it not need to reboot? Keep in mind if your system does hotplug well, you might not need to; but other systems will. I would expect we should be handling it in much the same way we handle hard_reset().

Reply:
Good point. Note added to description, also added a warning with confirmation, same as hard_reset() command.